### PR TITLE
Typography Design: Added some details for the typography section of the design handbook.

### PR DESCRIPTION
**Pull request by mapk**
Our typography page wasn't clear which fonts we are using, so I added some details around that to help clarify for contributors.

It now looks like this:
screen shot 2017-09-01 at 4 07 36 pm

### CODE REVIEW

**Comment by folletto** [comment]
I feel that thing needs some design work... but at first it's more important to have the addition you did in. ;)

**Comment by folletto** [comment]
There's just an eslint error to fix before merging:

> 78:1  error  Line 78 exceeds the maximum line length of 140  max-len

**Comment by mapk** [comment]
@folletto Can you help me out here? I opened the `./node_modules/.bin/eslint-eslines` file and I only have 59 lines of code (the error is on line 78?). I'm not sure how to troubleshoot this one. Does this file get compiled with more code during the process?

File: client/devdocs/design/typography.jsx [hunk 1]
```diff
@@ -75,6 +75,7 @@ export default React.createClass( {
 						<a href="/devdocs/design/typography">Typography</a>
 					</h2>
 					<h3>Interface Typography</h3>
+					<p>We use system fonts (<code>-apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell", "Helvetica Neue", sans-serif</code>) which improve the page rendering speed.</p>
```
Comment (folletto) [inline client/devdocs/design/typography.jsx]: It's this line :)

**Comment by apeatling** [comment]
@mapk & @folletto Can we merge this?

**Comment by mapk** [comment]
Oh yeah! I'll merge it now.

**Pull request merged**